### DISCUSSION TITLE
LDAP authentication/authorization

### DISCUSSION THREAD

**Discussion by git-syeda** [arkmq-org/arkmq-org-broker-operator#682] [Q&A] [answered]
Hi team,
Could you please let us know is there a way that we can set up LDAP authentication on artemis MQ via **CRD's**. As per artemiscloud.io documentation, we are unable to find any CRD that would help achieve this.

**Comment by gtully** [comment, marked as answer]
the entire jaas config can be specified from a secret. see: https://github.com/artemiscloud/activemq-artemis-operator/blob/16f211fa65203b82c4513b3440aaae0abacfdf69/docs/help/operator.md?plain=1#L833

You can reference the Artemis documentation to get a handle on the jaas LDAP login modules specifics.
